React/Modules/RCTDimensionManager.m

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "RCTDimensionManager.h"
+#import "RCTBridge.h"
+#import "RCTEventDispatcher.h"
+#import "RCTUtils.h"
+
+static NSDictionary *RCTCurrentDimensions()
+{
+    static NSDictionary *dimensions;
+    
+    CGSize frameSize = [UIScreen mainScreen].applicationFrame.size;
+    if ((NSFoundationVersionNumber <= NSFoundationVersionNumber_iOS_7_1)
+            && UIInterfaceOrientationIsLandscape([UIApplication sharedApplication].statusBarOrientation)) {
+        frameSize = CGSizeMake(frameSize.height, frameSize.width);
+    }
+    
+    dimensions = @{
+        @"width": [NSNumber numberWithFloat:frameSize.width],
+        @"height": [NSNumber numberWithFloat:frameSize.height]
```

Comment:
@pjjanak The syntax @(someFloatValue) is a shorthand in Objective-C for [NSNumber numberWithFloat:someFloatValue].